Microsoft.Toolkit.Uwp.UI.Controls/ImageCropper/ImageCropper.Logic.cs

```diff
@@ -20,10 +20,19 @@ public partial class ImageCropper
         /// </summary>
         private void InitImageLayout()
         {
-            _restrictedCropRect = new Rect(0, 0, Source.PixelWidth, Source.PixelHeight);
-            var maxSelectedRect = _restrictedCropRect;
-            _currentCroppedRect = KeepAspectRatio ? GetUniformRect(maxSelectedRect, UsedAspectRatio) : maxSelectedRect;
-            UpdateImageLayout();
+            if (Source != null)
+            {
+                _restrictedCropRect = new Rect(0, 0, Source.PixelWidth, Source.PixelHeight);
+                _currentCroppedRect = KeepAspectRatio ? GetUniformRect(_restrictedCropRect, UsedAspectRatio) : _restrictedCropRect;
+                UpdateImageLayout();
+            }
+            else
+            {
+                _currentCroppedRect = Rect.Empty;
```

Comment:
Can `_currentCroppedRect ` be set to empty in the set method in `Source` property?

Reply:
_currentCroppedRect does not affect functionality even if it is not reset to the initial value, so I removed the code.